src/registrar/templates/application_dotgov_domain.html

```diff
@@ -1,58 +1,83 @@
-  <ul class="usa-list">
-    <li>Be available </li>
-    <li>Be unique </li>
-    <li>Relate to your organization’s name, location, and/or services </li>
-    <li>Be clear to the general public. Your domain name must not be easily confused with other organizations.</li>
-  </ul>
+{% load static %}
+{% load field_helpers %}
+
+{% block form_instructions %}
+  <p>Before requesting a .gov domain, <a href="{% url 'todo' %}">please make sure it
+  meets our naming requirements.</a> Your domain name must:
+    <ul class="usa-list">
+      <li>Be available </li>
+      <li>Be unique </li>
+      <li>Relate to your organization’s name, location, and/or services </li>
+      <li>Be clear to the general public. Your domain name must not be easily confused
+      with other organizations.</li>
+    </ul>
   </p>
 
-  <p>Note that <strong>only federal agencies can request generic terms</strong> like vote.gov.</p>
+  <p>Note that <strong>only federal agencies can request generic terms</strong> like
+  vote.gov.</p>
 
-  <p>We’ll try to give you the domain you want. We first need to make sure your request meets our requirements. We’ll work with you to find the best domain for your organization.</p>
+  <p>We’ll try to give you the domain you want. We first need to make sure your request
+  meets our requirements. We’ll work with you to find the best domain for your
+  organization.</p>
 
   <p>Here are a few domain examples for your type of organization.</p>
   <div class="domain-example">
     {% include "includes/domain_example.html" %}
   </div>
-</div>
+{% endblock %}
 
-<form id="step__{{steps.current}}" class="usa-form usa-form--large" method="post" novalidate>
-  <h2>What .gov domain do you want?</h2>
-  {# TODO: aria-describedby to associate these instructions with the input! #}
-  <p id="domain_instructions">After you enter your domain, we’ll make sure it’s available and that it meets some of our naming requirements. If your domain passes these initial checks, we’ll verify that it meets all of our requirements once you complete and submit the rest of this form.</p>
 
+{% block form_fields %}
   {% csrf_token %}
-  {% input_with_errors forms.0.requested_domain www_gov=True %}
-  <button type="button" class="usa-button">Check availability</button>
 
-  <h2>Alternative domains</h2>
+  {{ forms.0.management_form }}
+
+  <fieldset class="usa-fieldset">
+    <legend>
+      <h2>What .gov domain do you want?</h2>
+    </legend>
+
+    <p id="domain_instructions">After you enter your domain, we’ll make sure it’s
+    available and that it meets some of our naming requirements. If your domain passes
+    these initial checks, we’ll verify that it meets all of our requirements once you
+    complete and submit the rest of this form.</p>
+
+    {% with attr_aria_describedby="domain_instructions domain_instructions2" %}
+      {% with www_gov=True attr_validate="domain" %}
```

Comment:
**suggestion (non-blocking)**: I generally loathe HTML comments but since the `validate="domain"` connects up to distant JS code, maybe we could use a template comment here to highlight that this is the important attribute?

Reply:
Commented.